cfme/tests/infrastructure/test_scvmm_specific.py

```diff
@@ -13,9 +13,10 @@ def pytest_generate_tests(metafunc):
 
 
 @pytest.mark.meta(blockers=[1178961])
-def test_no_dvd_ruins_refresh(provider_mgmt, provider_crud, small_template):
+def test_no_dvd_ruins_refresh(provider_mgmt, provider_crud, provider_data, small_template):
     with provider_mgmt.with_vm(
-            small_template, vm_name="test_no_dvd_{}".format(fauxfactory.gen_alpha())) as vm_name:
+            small_template, vm_name="test_no_dvd_{}".format(fauxfactory.gen_alpha()),
+            host_group=provider_data["host_group"]) as vm_name:
```

Comment:
Looking at the yaml, `host_group` is in `provider_data['provisioning_data']`. I think this passed PRT because it was skipped, but when the blockers are removed it will probably fail.

If `host_group` is required for these tests, I'd prefer to see them come in as argavlues in test generation so that we can skip the tests if the required provisioning data is missing.